includes/class-ajax.php

```diff
@@ -1133,7 +1133,7 @@ public function import_form() {
 
         $this->check_admin();
 
-        $the_file = isset( $_FILES['importFile'] ) ? sanitize_text_field( wp_unslash( $_FILES['importFile'] ) ) : false;
+        $the_file = isset( $_FILES['importFile'] ) ? $_FILES['importFile'] : false;
```

Comment:
@webacetechs why was the sanitation removed?

Reply:
$_FILES['importFile']  returns an array and  "wp_unslash" removes the " \ " from the path which leads a wrong path for a file and "sanitize_text_field" use to sanitize the string not an array or array of object which cause to return empty instead of an array.